src/lib/artworkToSubmissionMapping.ts

```diff
@@ -0,0 +1,70 @@
+const ValidSubmissionCategories = [
+  "ARCHITECTURE",
+  "DESIGN_DECORATIVE_ART",
+  "DRAWING_COLLAGE_OR_OTHER_WORK_ON_PAPER",
+  "FASHION_DESIGN_AND_WEARABLE_ART",
+  "INSTALLATION",
+  "JEWELRY",
+  "MIXED_MEDIA",
+  "PAINTING",
+  "PERFORMANCE_ART",
+  "PHOTOGRAPHY",
+  "PRINT",
+  "SCULPTURE",
+  "TEXTILE_ARTS",
+  "VIDEO_FILM_ANIMATION",
+]
+
+export const artworkToSubmissionCategory = (category: string) => {
+  if (!category) return
+
+  // replace all spaces, commas, and slashes with underscores, remove double underscores, and uppercase
+  const normalizedArtworkCategory = category
+    .replace(/ /g, "_")
+    .replace(/,/g, "_")
+    .replace(/\//g, "_")
+    .replace(/\_\_/g, "_")
+    .toUpperCase()
+
+  if (ValidSubmissionCategories.includes(normalizedArtworkCategory)) {
+    return normalizedArtworkCategory
+  }
+
+  return "OTHER"
+}
+
+// artwork is an object taken from the Gravity API (artworkLoader)
+export const artworkToSubmissionMapping = (artwork) => {
```

Comment:
I created a helper here [getInitialSubmissionFormValuesFromArtwork](https://github.com/artsy/eigen/blob/d87a2641244c4c65f853fa47a75796f7915a3975/src/app/Scenes/SellWithArtsy/ArtworkForm/Utils/getInitialSubmissionValuesFromArtwork.ts#L12) for some mapping

Reply:
thanks @MounirDhahri. The logic is more or less the same, but the difference is that your code is backed by GraphQL response, and our code uses raw gravity response (artworkLoader). I've checked if our code misses any fields that your code has and I added missing `signature` field.

Reply:
perfect - thanks